### ISSUE TITLE
Simplify Simplero Sales Page for Free Stuff - remove 'Billing Information' and '$'

### ISSUE THREAD

**Issue by janejacksoncareers** [Simplero/Roadmap#180]
Hi - In Simplero I have a number of free items but there is still 'billing info' and it looks, to the reader, like payment must be made. This may put people off because other landing pages allow you to just put in Name and Email address and then 'Sign Me Up!' which is much simpler and no barrier to entry.   

What we have currently in Simplero (for free and paid stuff) is this:
![screen shot 2017-11-07 at 5 16 59 pm](https://user-images.githubusercontent.com/33444940/32480938-55dc3998-c3e5-11e7-94f7-bd374e280130.png)

I doesn't need to have the 'billing information' and 'Your Payment' $0.0 field if it's free.  What would be better is something like this:
![screen shot 2017-11-07 at 5 23 25 pm](https://user-images.githubusercontent.com/33444940/32480997-9439ab4e-c3e5-11e7-843b-0832511b59a1.png)

It would put Simplero landing pages on more of a par with Leadpages who have lovely designs (and very simple). 

Also, if you could remove the sidebar on the sales page that would help readers to focus on the ONE page without unnecessary columns. The simpler the better for landing pages especially when no payment is expected.

Many thanks!

**Comment by calvincorreli** [comment]
Hi Jane

This is a checkout form, not a landing page.

Can't you do what you want with an actual landing page, and a List instead of a Product?

**Comment by janejacksoncareers** [comment]
Yes, but then in Simplero the landing page will take us to the checkout form/freebie checkout form.  Is it possible for the landing page to take you straight to a 'thank you for signing up' message and then the person gets the product without having to type in their email and details again? I may be missing something but I think that's what happens with the Simplero landing pages? Many thanks!

**Comment by calvincorreli** [comment]
Can you try reaching out to our support people using the HELP link, I'm sure they can help you figure this out.

**Comment by janejacksoncareers** [comment]
Ok Thanks Calvin!  And I hope you had a good Christmas and a wonderful 2018 coming up!
